userspace/libscap/scap.h

```diff
@@ -323,6 +324,7 @@ typedef enum scap_ifinfo_type
 */
 typedef struct scap_ifinfo_ipv4
 {
+	// NB: new fields must be appended
```

Comment:
Since we write the entire struct and don't serialize members field by field, I don't think appending will work fine unless we rework the way data is written to make it similar to the other blocks (also for the ipv6 struct). I hope what I am trying to say is clear.

Reply:
I'm probably missing something, but why isn't the same thing, since these `struct` are packed?

Reply:
What I meant is that we write the entire struct but we write it truncated to account for the extra unused space of `ifname[SCAP_MAX_PATH_SIZE]`, so nothing can be appended since `ifname` must always be the last member of the struct, unless we change the way the struct is written to a field by field basis.